tools/static-assets/scaffolds-ts/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./collection";
```

Comment:
maybe we should adjust code style for using one type of quotes for import/export? single or double?